<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>530</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TranDucTrungKien</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>C:/Users/kien0/OneDrive - zxcvb/Pictures/siu.gif</normaloff>C:/Users/kien0/OneDrive - zxcvb/Pictures/siu.gif</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="labelTen">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>751</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(20, 20, 20);
color: rgb(255, 85, 255);</string>
    </property>
    <property name="text">
     <string>Con Cho Thanh Thinh</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>110</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Nhập N:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditNhapN">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>110</y>
      <width>461</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelKetQua">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>250</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Kết quả:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditKetQua">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>250</y>
      <width>461</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonGiai">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>370</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Giải</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>C:/Users/kien0/OneDrive - zxcvb/Pictures/banhrang.webp</normaloff>C:/Users/kien0/OneDrive - zxcvb/Pictures/banhrang.webp</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonTiep">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>370</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Tiếp</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>C:/Users/kien0/OneDrive - zxcvb/Pictures/cuctay.png</normaloff>C:/Users/kien0/OneDrive - zxcvb/Pictures/cuctay.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonExit">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>370</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Thoát</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>C:/Users/kien0/OneDrive - zxcvb/Pictures/dauX.png</normaloff>C:/Users/kien0/OneDrive - zxcvb/Pictures/dauX.png</iconset>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
